openapi/components/schemas/Common/CommonSubscriptionOrder.yaml

```
@@ -59,6 +60,36 @@ allOf:
         example: null
         allOf:
           - $ref: ../Invoices/InvoiceTimeShift.yaml
+      invoiceIssueTimeShift:
+        type: string
+        description: >
+          The calculation instruction of billing time.
+
+
+          This is used in conjunction with the **service period anchor** to calculate
+          the time the invoice is issued.
+
+
+          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
+        nullable: true
+        default: '-P00Y00M00DT00H00M00S'
+        example: '-P7D'
+      invoiceDueTimeShift:
+        type: string
+        description: >
+          The calculation instruction of due time.
+
+
+          This is used in conjunction with the issue time to calculate due time of invoice.
+
+
+          The chronology of due time shift is always **after**.
+
+
+          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
```

```suggestion
          Time period between the invoice issue time shift and the invoice due time shift in **ISO-8601** format.
          Period must be positive.
```
The term `chronology` seems not relevant to this value anymore

---

Needs a better explanation still. I have an idea...

---

Thanks for suggestions, updated